integration_test-process.txt

```diff
@@ -125,23 +144,73 @@ After installing the test package with "zypper in"
   + Cloud based RMT server is the target
 - registercloudguest --clean
 
-Migration testing
+Migration testing and LTSS testing
 SLE 15
 - Start a PAYG instance 2 SPs back from the current one
 - Check the instance is properly registered
 - zypper up
   + Expected to succeed
-- With the new cloud-regionsrv-client code in place run "zypper migration"
+  With the new cloud-regionsrv-client code in place
+- registercloudguest --clean
+- registercloudguest -r LTSS_REG_CODE
+  + Expected to fail because the underlying system base product is not
+    registered
+- registercloudguest
+  + no error
+  + sucees message on stdout
+  + $? is 0
+- registercloudguest -r LTSS_REG_CODE
+  + zypper lr | grep LTSS has LTSS repositories
+  + zypper ref
+  + zypper up
+    + Expected to have new updates due to the addition o fthe LTSS repos
+  + source /etc/profile.local
+  + systemctl start docker.service
+  + systemctl status docker.service
+    - Service should be running
+  + docker pull suse/ltss/sle$VERSION/bci-base
+    - Should pull the container from registry-$FRAMEWORK.susecloud.net
+  + systemctl stop docker.service
+  + zypper in podman
+  + podman pull suse/ltss/sle$VERSION/bci-base
+- zypper migration
+  + Should fail because LTSS repos are active, and possibly othre modules
+  + Use the command presented to remove the product/modules
+- zypper migration
   + Migrate to the highest available SP
   + Expected to succeed
 - Start a BYOS instance 2 SPs back from the current one
+  With the new cloud-regionsrv-client code in place
 - registercloudguest -r XXX
   + no error
   + sucees message on stdout
 - Add the Public Cloud Module if it is not setup
 - zypper up
   + Expected to succeed
-- With the new cloud-regionsrv-client code in place run "zypper migration"
+- registercloudguest -r LTSS_REG_CODE
+  + zypper lr | grep LTSS has LTSS repositories
+  + zypper ref
+  + zypper up
+    + Expected to have new updates due to the addition o fthe LTSS repos
+  + source /etc/profile.local
+  + systemctl start docker.service
+  + systemctl status docker.service
+    - Service should be running
+  + docker pull suse/ltss/sle$VERSION/bci-base
+    - Should pull the container from registry-$FRAMEWORK.susecloud.net
+  + systemctl stop docker.service
```

Comment:
can we not run podman while docker is running ? I dont remember if it raised a socket error

Reply:
I am not sure if the docker daemon and podman can run at the same time but that has nothing to do with us. SO stopping docker avoids the question/problem ;)